### DISCUSSION TITLE
Helm Chart Release ?

### DISCUSSION THREAD

**Discussion by dntosas** [apache/airflow#37105] [General] [answered]
### Official Helm Chart version

1.11.0 (latest released)

### Apache Airflow version

2.8.1

### Kubernetes Version

1.28

### What happened

Hey team! 👋 

Any chance we get a new Helm chart release soon?

### How to reproduce

-

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment, marked as answer]
I suggest you avoid creating issues for questions. If you look at PRs, you will find the release is being prepared. Can we count you helping with testing release candidate ? Please subscribe to the devlist and we will count on your "yes tested it,  looks good" when we ask for it.

Can we count on it @dntosas ?
